### DISCUSSION TITLE
Can't Download immediately deleted by 360 total security.

### DISCUSSION THREAD

**Discussion by Leomist** [Tyrrrz/DiscordChatExporter#1216] [Q&A]
My virus protection software keeps flagging it as a virus. 
Why is it flagged as a virus?

**Comment by ghost** [comment]
Not a virus.

**Comment by Technetium1** [comment]
It's a false positive, here's [some VirusTotal confirmation](https://www.virustotal.com/gui/file/0ab3028d78e7b97bd557f4e3a4c4687f32609962592cd1d7dcda073c86b4e1a4). Please reconsider using Quihoo software, you'd even be better off with Windows defender or almost anything else. Their software has a history of truly _**awful**_ things, see for yourself: https://www.google.com/search?q=backdoor+in+quihoo+software
